packages/as-proba/README.md

```diff
@@ -1,5 +1,86 @@
 # as-proba
 
-This package provides mathematic and probabilistic classes and functions.
+**MassaLabs** as-proba library.
 
-- [`as-proba documentation`](https://as-proba.docs.massa.net)
\ No newline at end of file
+This library provides useful *AssemblyScript* objects and functions :
+
+- **Binomial** in an object that represent a binomial distribution. It can be used to compute probabilities of a given event and generate random samples
+
+- Useful methods to compute **combinations** an **partial permutations** 
+
+- **randomInt**, a method to generate a random number between given limits
+
+- **Sampler**, an object wich allows you to generate samples based on the probability distribution you built
+
+The complete documentation of all available functions and objects can be found here:
+
+- [`as-proba documentation`](https://as-proba.docs.massa.net)
+
+## Warning
+**Non-secure random functions are used here.**
+
+Using non-secure random functions is the only way to have onchain randomness but **these functions generates pseudo-random values (predictable and repeatable randomness), it is NOT a real randomness**.
+Therefore it is easier for attackers to predict the output and exploit weaknesses in your smart contract.
+
+## Install
+
+Packages are independant you can choose to install what you need
+
+```sh
+npm i --save-dev @massalabs/as-proba
+```
+
+## Development guide
```

Comment:
This should be in contributing. have a look at how things aresplitted in massa-as-sdk between the read me an the contributing